packages/data-encoding/data-encoding.0.6/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+maintainer: "contact@nomadic-labs.com"
+authors: [ "Nomadic Labs" ]
+homepage: "https://gitlab.com/nomadic-labs/data-encoding"
+bug-reports: "https://gitlab.com/nomadic-labs/data-encoding/issues"
+dev-repo: "git+https://gitlab.com/nomadic-labs/data-encoding.git"
+license: "MIT"
+depends: [
+  "ocaml" { >= "4.10" }
+  "dune" { >= "2.0" }
+  "ezjsonm"
+  "zarith" {>= "1.4"}
+  "zarith_stubs_js"
+  "hex" {>= "1.3.0"}
+  "json-data-encoding" { = "0.11" }
+  "json-data-encoding-bson" { = "0.11" }
+  "alcotest" { with-test }
+  "crowbar" { >= "0.2" & with-test }
+  "ppx_expect" { with-test }
+  "either"
+  "ppx_hash"
+  "ocamlformat" { = "0.20.1" & with-doc } # not technically a doc dep; modify when with-dev becomes available
+  "odoc" { with-doc }
+  "md2mld" { with-test } # not technically a test dep; modify when https://github.com/ocurrent/ocaml-ci/issues/264 is fixed
```

Comment:
I’m guessing those depedendencies aren’t actually required outside of a personal dev setup?
```suggestion
  "odoc" { with-doc }
```

Reply:
Is that what `with-tools` is for? From the opam doc

>     `with-test`: only true if tests have been enabled for this specific package
>     `with-doc`: similarly for documentation
>     `with-tools`: similarly for developer tools

That's very terse documentation so I'm not 100% sure. Not sure what it means to _enable_ developer tools.